library/hls/src/main/java/com/google/android/exoplayer2/source/hls/HlsSampleStreamWrapper.java

```diff
@@ -146,6 +146,8 @@
   private @MonotonicNonNull Format upstreamTrackFormat;
   @Nullable private Format downstreamTrackFormat;
   private boolean released;
+  private boolean interrupted;
```

Comment:
I removed this boolean and "merged" it with the other variable by using POSITION_UNSET if no cancellation is pending.